app/views/layouts/application.html.erb

```diff
@@ -7,6 +7,13 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
+  <h1>DBCbook</h1>
+  <%if logged_in?%>
+  <%= link_to "Home", "/"%> |
```

Comment:
Think about indenting the content of if / each blocks to make the view easier to read.